### PR TITLE
Add orange.fr

### LINKED ISSUE #2257: Add Orange.fr

### PR DESCRIPTION
**Pull request by divayprakash**
Closes #2257

### CODE REVIEW

**Review by Carlgo11 — APPROVED** [review]

File: _data/utilities.yml [hunk 1]
```diff
@@ -150,6 +150,14 @@ websites:
       img: optus.png
       tfa: No
 
+    - name: Orange
+      url: https://orange.fr/
```
Comment (Jawshy) [inline _data/utilities.yml]: The current `url` redirects to a HTTP page for me. @divayprakash, please use `https://www.orange.fr/portail` instead.

**Review by Jawshy — CHANGES_REQUESTED** [review]

File: _data/utilities.yml (hunk 1 above)
Comment (divayprakash) [reply _data/utilities.yml]: Fixed in ca69c48

Comment (Jawshy) [reply _data/utilities.yml]: Needs the `www.`.

Comment (divayprakash) [reply _data/utilities.yml]: Fixed in 04bfa7c

**Review by Jawshy — APPROVED** [review]

**Pull request merged**